package.json

```diff
@@ -65,7 +65,7 @@
     "tslint": "^5.16.0"
   },
   "scripts": {
-    "test": "npm run build && mocha -t 5000 --require source-map-support/register test",
+    "test": "npm run build && mocha -t 5000 --require source-map-support/register dist/test",
     "debug": "ts-node --inspect=19248 --debug-brk typescript-json-schema-cli.ts",
     "run": "ts-node typescript-json-schema-cli.ts",
     "build": "tsc -p .",
```

Comment:
needs a `prepublishOnly `

Reply:
What do you mean? Do you want to run the build on `prepublishOnly`?

Reply:
We want to run the build. See https://github.com/vega/ts-json-schema-generator/blob/master/package.json#L61

Reply:
Added